### PR TITLE
sys-libs/compiler-rt-sanitizers: fix 4.0.1 on glibc 2.26

### PR DESCRIPTION
**Pull request by alyst**
Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

**Comment by mgorny** [comment]
Thank you for your pull request. However, I'm not going to put a patch for an old version in FILESDIR. Instead, I'm going to put it into my devspace and update the ebuild myself. For the future, please also remember to lower keywords to ~arch when adding new revision.

File: sys-libs/compiler-rt-sanitizers/compiler-rt-sanitizers-4.0.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		sys-libs/compiler-rt:${SLOT} )
+	${PYTHON_DEPS}"
+
+S=${WORKDIR}/compiler-rt-${PV/_/}.src
+
+# least intrusive of all
+CMAKE_BUILD_TYPE=RelWithDebInfo
+
+PATCHES=(
+	"${FILESDIR}"/sanitizer_fix_glibc.patch
```
Comment (mgorny) [inline sys-libs/compiler-rt-sanitizers/compiler-rt-sanitizers-4.0.1-r1.ebuild]: Also, you have added a different file here.

Comment (alyst) [reply sys-libs/compiler-rt-sanitizers/compiler-rt-sanitizers-4.0.1-r1.ebuild]: oops, sorry, forgot to copy the updated one from `local/portage` into github tree. but that should be the only difference and the patch contents should be identical

Comment (mgorny) [reply sys-libs/compiler-rt-sanitizers/compiler-rt-sanitizers-4.0.1-r1.ebuild]: I actually see two patches on the relevant bug report, so I'm going to take both.

**Comment by alyst** [comment]
> I'm going to put it into my devspace and update the ebuild myself.

No problem. I'm just an "obligate" gentoo prefix user that tinkers around random issues that `emerge ... @world` brings.
Unfortunately, I don't have resources to become fluent with multiple Gentoo policies.

Thanks for taking it over. Would your updated ebuild find its way to this repo or I should plug some overlay?

**Comment by mgorny** [comment]
It should be there already.

**Pull request closed without merging**